server/config/store_config.go

```diff
@@ -192,99 +200,64 @@ func (c *StoreConfig) Clone() *StoreConfig {
 
 // StoreConfigManager is used to manage the store config.
 type StoreConfigManager struct {
-	config atomic.Value
-	source Source
+	client      *http.Client
+	persistOpts *PersistOptions
 }
 
 // NewStoreConfigManager creates a new StoreConfigManager.
-func NewStoreConfigManager(client *http.Client) *StoreConfigManager {
-	schema := "http"
-	if netutil.IsEnableHTTPS(client) {
-		schema = "https"
-	}
-
+func NewStoreConfigManager(client *http.Client, persistOpts *PersistOptions) *StoreConfigManager {
```

Comment:
Do we still need the store config manager?

Reply:
Removed. PTAL.